### PR TITLE
Upgrade to latest release

### PR DESCRIPTION
**Pull request by unixsheikh**
I have upgraded the terminal package and have successfully build and tested it on x64.

### CODE REVIEW

**Comment by maxice8** [comment]
Use the proper commit style.

`<pkgname>: update to <version>.`

File: srcpkgs/terminator/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 pkgname=terminator
-version=1.0
-revision=2
+version=1.91
+revision=1
 build_style=python2-module
 hostmakedepends="python intltool"
 makedepends="vte-python"
-depends="desktop-file-utils vte-python python-psutil"
+depends="desktop-file-utils vte-python python-psutil vte3"
```
Comment (maxice8) [inline srcpkgs/terminator/template]: Seems like something to be added to makedepends and be found directly by shlib-depends

Comment (unixsheikh) [reply srcpkgs/terminator/template]: Yes, sorry, new to Void :)

Comment (maxice8) [reply srcpkgs/terminator/template]: no need to actually change, terminator is a python terminal so we can't use shlib depends.

**Comment by Gottox** [comment]
Thanks for your contribution!

Unfortunately terminator 1.91 is the unfinished development version and I can't see a good reason for merging it. I'd like to wait till the actual 2.0 release is out.

**Comment by unixsheikh** [comment]
There is a bug in the current version which messes up neovim and possibly also vim (I haven't tested that). The new version fixes that problem.

void-terminator

**Pull request closed without merging**